modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc

```diff
@@ -0,0 +1,58 @@
+:description: How to use Cypher to manage property-based access control on graphs.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE ROLE regularUsers;
+----
+////
+
+[role=enterprise-edition aura-db-enterprise]
+[[property-based-access-control]]
+= Property-based access control
+
+It is possible to create `READ` privileges that are based on properties on nodes.
```

Comment:
using `READ` in this format makes me think it only applies to the READ privilege and not all the read privileges aka READ, TRAVERSE and MATCH. I think this should be make more clear in this whole paragraph